src/main/java/servece/impl/ProcessorServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package servece.impl;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.ProcessorService;
+import service.handler.OperationHandler;
+
+public class ProcessorServiceImpl implements ProcessorService {
+
+    @Override
+    public Map<String, Integer> processOnData(List<FruitTransaction> transactions,
+                                              Map<FruitTransaction.Operation,
+                                                      OperationHandler> operation) {
+        Map<String, Integer> fruitBalance = new HashMap<>();
```

Comment:
this shouldn't be just a local field, refactor your architecture
storage should be a separate class